tests/cri-o/run-tests.sh

```diff
@@ -33,6 +33,8 @@ sed -i -e 's|@test "privileged ctr device add" {|@test "privileged ctr device ad
 sed -i -e 's|@test "ctr device add" {|@test "ctr device add" {\nskip\n|g' test/*.bats
 sed -i -e 's|@test "privileged ctr -- check for rw mounts" {|@test "privileged ctr -- check for rw mounts" {\nskip\n|g' test/*.bats
 sed -i -e 's|@test "kubernetes pod terminationGracePeriod passthru" {|@test "kubernetes pod terminationGracePeriod passthru" {\nskip\n|g' test/*.bats
+sed -i -e 's|@test "ctr create with non-existent command" {|@test "ctr create with non-existent command" {\nskip\n|g' test/*.bats
+sed -i -e 's|@test "ctr create with non-existent command \[tty\]" {|@test "ctr create with non-existent command \[tty\]" {\nskip\n|g' test/*.bats
```

Comment:
maybe it is better if we fix the test upstream for CRI-O.

I think the test was broken with 7a5b3a29062082f6d755a61a487843f604422cc9

Reply:
I am fine to skip it for now though, we can revert once it works in CRI-O.

Reply:
Sure.I'm just waiting to check if CI passes now. I agree we must fix this in upstream `cri-o` as well.

Reply:
it seems to work with upstream, but not v1.18.6.

Should we try with origin/master and see what happens?

Reply:
@giuseppe are you able to build from `master` ? Both for me and CI `make` fails with 

```console
$ make
mkdir -p "/home/arajan/work/cri-o/_output/src/github.com/cri-o"
ln -s "/home/arajan/work/cri-o" "/home/arajan/work/cri-o/_output/src/github.com/cri-o/cri-o"
touch "/home/arajan/work/cri-o/_output/.gopathok"
go build -trimpath  -ldflags '-s -w -X github.com/cri-o/cri-o/internal/pkg/criocli.DefaultsPath="" -X github.com/cri-o/cri-o/internal/version.buildDate='2022-01-31T11:56:06Z' -X github.com/cri-o/cri-o/internal/version.gitCommit=1801aea989e811f5b055b771962aa7256ef0168b -X github.com/cri-o/cri-o/internal/version.gitTreeState=dirty ' -tags "containers_image_ostree_stub      containers_image_openpgp seccomp selinux " -o bin/crio github.com/cri-o/cri-o/cmd/crio
# sigs.k8s.io/json/internal/golang/encoding/json
vendor/sigs.k8s.io/json/internal/golang/encoding/json/encode.go:1249:12: sf.IsExported undefined (type reflect.StructField has no field or method IsExported)
vendor/sigs.k8s.io/json/internal/golang/encoding/json/encode.go:1255:18: sf.IsExported undefined (type reflect.StructField has no field or method IsExported)
make: *** [Makefile:175: bin/crio] Error 2
```

and here is how it fails in CI: https://github.com/containers/crun/runs/5002497227?check_suite_focus=true

Reply:
I think you just need go 1.17

Reply:
@haircommander Thanks a lot. Yeah it seems we need to bump CI to `go1.17`, I'll do it thanks.